lib/storage/package.json

```diff
@@ -23,11 +23,11 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
+    "@aws-sdk/abort-controller": "3.4.1",
```

Comment:
shouldn't be 3.5.0 ?

Reply:
The newest version of abort controller is 3.4.1. https://www.npmjs.com/package/@aws-sdk/abort-controller. We only bump packages when needed because otherwise we end up publishing a bunch of packages with zero diffs from previous versions. 

This might be a bit confusing since the versions aren't exactly following one another. But we do also have an @latest tag to alleviate it.


**eg.**  its entirely possible in the future to only have versions `3.6.0` and `3.9.0` of ` "@aws-sdk/abort-controller"` published. Without the minor versions in between them (`3.7.0`, `3.8.0`).